lib/helpers/streamChunksOfCombinedSourceMap.js

```diff
@@ -153,7 +153,7 @@ const streamChunksOfCombinedSourceMap = (
 							}
 							const name = nameIndexValueMapping[nameIndex];
 							const originalName =
-								innerOriginalLine <= originalSourceLines.length
+										(originalSourceLines !== null && innerOriginalLine <= originalSourceLines.length)
```

Comment:
Wrong indentation, and I think the parenthesis are not required.

Reply:
How about [e3a3701](https://github.com/webpack/webpack-sources/pull/121/commits/e3a3701ead2951bfeb8690ee2ca78b4f3b25e14c)